### DISCUSSION TITLE
[월드 1.1.5] TouchPad 방향을 이벤트 받아오기 (Joystick, 조이스틱)

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#593] [5. Q&A]
월드 1.1.5

#583
#589

확인했습니다만 다른 문제인듯 합니다
이벤트연결시 스크립트명과 함수명이 안보이는 문제입니다

TouchPad의 방향을 이벤트로 받아오려고 합니다 
![image](https://user-images.githubusercontent.com/50188795/162013293-9d7b6ba5-e409-46de-abb4-26517e5d5b98.png)

다음과 같이 프리팹을 가져왔습니다 
![image](https://user-images.githubusercontent.com/50188795/162013485-b547e56d-a71f-4f48-bb98-75609d61021e.png)

OnDragEvent 에 추가를 하고 
![image](https://user-images.githubusercontent.com/50188795/162013554-fc1f6592-f5c6-449c-bbe2-de5cb6349efa.png)

스크립트를 만들고 
![image](https://user-images.githubusercontent.com/50188795/162013651-8ee3a37e-f162-44f5-b48d-2a24b449fdd8.png)
![image](https://user-images.githubusercontent.com/50188795/162013688-d092053d-d22d-43ff-8c1d-87db7bc80c17.png)

Pad에 부착한뒤 
![image](https://user-images.githubusercontent.com/50188795/162013769-99ac45e3-2664-4292-9f4f-c5426582f061.png)
Assign해줬습니다 
![image](https://user-images.githubusercontent.com/50188795/162013882-1a97e78c-1ccd-45c8-b7b0-e9be3fda1fb7.png)

제가 만든 MyTouchPad의 함수는 왜 안보이는것일까요?
![image](https://user-images.githubusercontent.com/50188795/162013997-9a646fcd-0dc3-4693-8367-30eea983ccdc.png)
![image](https://user-images.githubusercontent.com/50188795/162014036-f258bc04-e930-4cbd-986b-202b20cefaa7.png)

지금 제가 하려는게 가능한가요?
가능하다면 어떻게 해야 하나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. Inspector를 이용하여 TypeScript 연결이 안되는 이슈는 인지하고 있으며 내부적으로 어떻게 해결할 지 고민을 하고 있는 단계입니다.
조금 불편하시겠지만 아래 가이드와 같이 TypeScript 내부에서 이벤트 연결을 하시면 동작은 가능한 점 참고 부탁드립니다.
https://studio.zepeto.me/guides/ui-events

**Reply by smilejsu82** [reply]
이벤트를 스크립트로 넣어도 에러가 납니다 
#613 확인 부탁 드려요
